pkg/kubelet/userns/userns_manager.go

```diff
@@ -270,7 +271,7 @@ func (m *UsernsManager) Release(podUID types.UID) {
 	m.lock.Lock()
 	defer m.lock.Unlock()
 
-	m.releaseWithLock(podUID)
+	m.releaseWithLock(klog.TODO(), podUID)
```

Comment:
Can you pass logger(or context) from the upper layer?

Reply:
Done~